### PR TITLE
Suppress warning in Combinatorial  Map for g++ >= 5

### PR DESCRIPTION
**Pull request by afabri**
Fixing [these warnings](https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.8-Ic-113/Combinatorial_map/TestReport_lrineau_Ubuntu-latest-GCC6-Release.gz)

### CODE REVIEW

File: Combinatorial_map/include/CGAL/Combinatorial_map_storages.h [hunk 1]
```diff
@@ -405,5 +411,9 @@ namespace CGAL {
 
 } // namespace CGAL
 
+
+#if  (BOOST_GCC >= 40800)
+ _Pragma("GCC diagnostic pop")
+#endif
 #endif // CGAL_COMBINATORIAL_MAP_H //
```
Comment (lrineau) [inline Combinatorial_map/include/CGAL/Combinatorial_map_storages.h]: The diagnostic options are pushed for gcc>=5.0 but popped for gcc>=4.8! That will probably do something strange for 4.8<=gcc<5.

File: Linear_cell_complex/include/CGAL/Linear_cell_complex_storages.h [hunk 2]
```diff
@@ -437,5 +443,8 @@ namespace CGAL {
 
 } // namespace CGAL
 
+#if  (BOOST_GCC >= 40800)
+ _Pragma("GCC diagnostic pop")
+#endif
 #endif // CGAL_LINEAR_CELL_COMPLEX_STORAGES_H //
```
Comment (lrineau) [inline Linear_cell_complex/include/CGAL/Linear_cell_complex_storages.h]: same here

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-115

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-116

**Comment by sloriot** [comment]
The warnings are [gone](https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.8-Ic-115/Combinatorial_map/TestReport_lrineau_Ubuntu-latest-GCC6-Release.gz)

**Pull request merged**